screens/CreateAccount.tsx

```
@@ -130,7 +129,7 @@ export default function CreateAccount() {
           accessibilityLabel="Password"
           accessibilityHint="Enter your password"
           returnKeyType="done"
-          onSubmitEditing={() => { handleAccountCreation }}
+        onSubmitEditing={() => handleAccountCreation()}
```

ESLint

[error] Expected indentation of 16 spaces but found 8.

(@stylistic/js/indent)